site-src/geps/gep-1424.md

```diff
@@ -0,0 +1,384 @@
+* Defining how multiple `Services` or `EndpointSlices` representing instances of a single "logical" service should present an identity for AuthN/AuthZ or be associated with each other beyond routing rules.
+* Defining how AuthZ should be implemented to secure East/West traffic between services.
+* Defining how [Policy Attachment](https://gateway-api.sigs.k8s.io/references/policy-attachment/) would bind to `HTTPRoute`, services or a mesh.
+* Defining how `HTTPRoutes` configured for East/West service mesh traffic management might integrate with North/South `Gateways`.
+    * This is a bit tricky in that it's effectively a form of delegation as described in [GEP-1058: Route Inclusion and Delegation](https://github.com/kubernetes-sigs/gateway-api/pull/1085), and is planned to be explored in a future GEP.
+* Handling East/West traffic outside the cluster (VMs, etc).
+
+## Introduction
+
+It is proposed that an application owner should configure traffic rules for a mesh service by configuring an `HTTPRoute` with a Kubernetes `Service` resource as a `parentRef`.
+
+This approach is dependent on both the "frontend" role of the Kubernetes `Service` resource as defined in [GEP-1324: Service Mesh in Gateway API](https://gateway-api.sigs.k8s.io/geps/gep-1324/#service) when used as a `parentRef` and the "backend" role of `Service` when used as a `backendRef`. It would use the Kubernetes DNS name to match traffic for meshes implementing "transparent proxy" functionality, but the `backendRef` endpoints would ultimately be used for the canonical IP address(es) to which traffic should be redirected by rules defined in this HTTPRoute. This approach leverages the existing points of extensibility within the Gateway API spec, and would not require introducing any API changes or new resources, only defining expected behavior.
+
+## API
+
+```
+metadata:
+  name: foo_route
+  namespace: store
+spec:
+  parentRefs:
+  - kind: Service
+    name: foo
+  rules:
+    backendRefs:
+    - kind: Service
+      name: foo
+      weight: 90
+    - kind: Service
+      name: foo_v2
+      weight: 10
+```
+
+In the example above, routing rules have been configured to direct 90% of traffic for the `foo` `Service` to the default "backend" endpoints specified by the `foo` `Service` [`selector`](https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service) field, and 10% to the `foo_v2` `Service`. This is determined based on the `ClusterIP` (for `Service`) and `ClusterSetIP` (for `ServiceImport`) matching, and for "transparent proxy" mesh implementations would match all requests to `foo.svc.cluster.local` (or arbitrary custom suffix, as the hostname is not specified manually) from within the same namespace, all requests to `foo.store.svc.cluster.local` from other namespaces, and all requests to `foo.store.svc.clusterset.local` for multicluster services, within the scope of the service mesh.
```

Comment:
nit: I think its worth saying "This means we are matching the service" then "services can be arbitrary things... if the are Service <behavior>, if ServiceImport <behavior>, else implementation specific".

Just to make it clear that it is one of those things that has arbitrary inputs but for the well-known types we have explicit behavior

Reply:
I guess we basically say that here, so not sure what my request is :-)